apps/chat_server/test/test_helper.exs

```diff
@@ -1,5 +1,5 @@
-{:ok, _} = Application.ensure_all_started(:ex_machina)
-
-Ecto.Adapters.SQL.Sandbox.mode(ChatServer.Repo, :manual)
+ExUnit.configure(exclude: [pending: true])
 
 ExUnit.start()
+
+Ecto.Adapters.SQL.Sandbox.mode(ChatServer.Repo, :manual)
```

Comment:
Thanks for adding a `pending` test pattern!

Could you also add back in the removed line:

```
{:ok, _} = Application.ensure_all_started(:ex_machina)
```

I think it got removed by accident when I merged in master. Appreciate it!